WooCommerce/Classes/POS/Presentation/ItemGridView.swift

```diff
@@ -21,11 +21,12 @@ struct ItemGridView: View {
             ScrollView {
                 LazyVGrid(columns: columns, spacing: 16) {
                     ForEach(viewModel.items, id: \.productID) { item in
-                        ItemCardView(item: item) {
-                            viewModel.addItemToCart(item)
-                        }
+                        ItemCardView(item: item)
```

Comment:
What do you think about having `ItemCardView` as a label part of a `Button`? We can set the button style but by default we would get touch feedback and accessibility out of the box.

Reply:
That's an interesting one: I did initially add it as a label to a button but the compiler decided it wouldn't compile it couldn't translate from `ItemCardView` to `View` when used as label. I tried today again and the compiler is happy with it 😂 

Updated on https://github.com/woocommerce/woocommerce-ios/pull/13069/commits/4946d34d4dce172502a8d5b04acb91813596c477